test/sanity/validate-modules/ignore.txt

```diff
@@ -0,0 +1,129 @@
+lib/ansible/modules/cloud/amazon/ec2_vpc_subnet.py E317
```

Comment:
The lines should be unique, no need for duplicates.

Reply:
Thanks, it didn't even occur to me that there may have been duplicates when I wrote out that file.